### PR TITLE
Update VAS_AutoBalance

### PR DESCRIPTION
**Pull request by milestorme**
Adding global announcer option, as requested

### CODE REVIEW

File: src/VASAutoBalance.cpp [hunk 1]
```diff
@@ -229,10 +229,11 @@ class VAS_AutoBalance_PlayerScript : public PlayerScript
         }
 
         void OnLogin(Player *Player)
-        {
-            if (enabled)
-                ChatHandler(Player->GetSession()).PSendSysMessage("This server is running a VAS_AutoBalance Module.");
-        }
+	{
```
Comment (Stoabrogga) [inline src/VASAutoBalance.cpp]: Please use blanks instead of tabs (4 blanks instead of 1 tab).

**Comment by milestorme** [comment]
if someone else who understands my dumb grammer would assist please.  the grammar police are here

**Comment by milestorme** [comment]
sorry barb im too dumb i dont know what he wants. closing will use my own version

**Comment by milestorme** [comment]
I copied that announcer from the updated skeleton module maybe you should bug them about spaces not tabs

**Comment by Stoabrogga** [comment]
I did not mean to offend you, but we should use a basic set of coding standards or else the code would look awful over time. It's not really that hard: http://www.azerothcore.org/wiki/Coding-Standards
```
We DO NOT ACCEPT tabs in code, you must convert tabs in 4 spaces ( you can change your IDE/editor settings, they must have this option, otherwise uninstall it )
```
I would have done this myself, but you deleted the repository. Cannot help here further.

**Comment by milestorme** [comment]
I copied it from skeleton module as barb asked , I’m not as smart as you guys I just wanted to share a working announcer , I don’t like keeping things for just myself I like everyone to benefit

**Comment by Stoabrogga** [comment]
I see. It's not about being "smart" (I don't think I'm smart), it's just about code style, nothing else. If I had known this earlier that you only wanted to share this upgrade I would have changed your PR. Some people are offended if you manipulate their PRs, so I'm careful about this.
Like you I also don't want to keep things for myself. That's the reason why I joined AC and will provide each and every improvement I implement on my own server. But I also had to learn that contributions to such a big project have to follow a basic set of rules.
If it is ok with you I'll create a new PR with your changes. Here's also the difference what this was all about:

With tabs in the "OnLogin" function:
```cpp
    public:
        VAS_AutoBalance_PlayerScript()
            : PlayerScript("VAS_AutoBalance_PlayerScript")
        {
        }

        void OnLogin(Player *Player)
	{
		if (sConfigMgr->GetBoolDefault("VASAutoBalanceAnnounce.enable", true)) {
			ChatHandler(Player->GetSession()).SendSysMessage("This server is running the |cff4CFF00VAS_AutoBalance |rmodule.");
		}
	}
```
With blanks:
```cpp
    public:
        VAS_AutoBalance_PlayerScript()
            : PlayerScript("VAS_AutoBalance_PlayerScript")
        {
        }

        void OnLogin(Player *Player)
        {
            if (sConfigMgr->GetBoolDefault("VASAutoBalanceAnnounce.enable", true)) {
                ChatHandler(Player->GetSession()).SendSysMessage("This server is running the |cff4CFF00VAS_AutoBalance |rmodule.");
            }
        }
```
At first glance the difference is not that big, but tabs are handled different in each editor and on top of that if you mix them with blanks the resulting code looks very ugly and unstructured. So it was decided to only use 4 blanks for indentation, which will look the same in each editor.

Oh, and thanks for the note about the skeleton module, I will look into this.

**Comment by Stoabrogga** [comment]
The skeleton module is ok, there are no tabs anywhere. Must have been your editor which replaces blanks with tabs when pasting text (some editors do that, sadly).

**Comment by BarbzYHOOL** [comment]
@milestorme we use this thing called configeditor too, it's a file that goes alongside a plugin to automatically adapt to the project's rules (tabs vs spaces for example)

There is a plugin for a lot of IDEs. Just install it and it should be fine :P

**Pull request closed without merging**